docs/managed-database-connection.md

```diff
@@ -0,0 +1,43 @@
+# Managed GCP PostgreSQL database connection
```

Comment:
maybe rename it to Configure Managed GCP PostgreSQL

Reply:
done